app/json_convert/src/main.cpp

```diff
@@ -0,0 +1,188 @@
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <csignal>
+#include <cstdio>
+#include <cstring>
+#include <iostream>
+#include <sstream>
+
+#include "lib/dls/dls.h"
+#include "lib/telemetry/TelemetryViewer.h"
+#include "lib/vcm/vcm.h"
+
+#define NUM_SIGNALS 5
+#define SERVER_IP "127.0.0.1"
+#define SERVER_PORT 5000
+
+
+using namespace dls;
+using namespace vcm;
+
+TelemetryViewer tlm;
+
+bool killed = false;
+
+const int SIGNALS[NUM_SIGNALS] = {SIGINT, SIGTERM, SIGSEGV, SIGFPE, SIGABRT};
+
+void err_handle(std::string message, MsgLogger *logger) {
+    std::string err_msg = "JSON_Convert: ";
```

Comment:
MsgLogger will actually append the class name / function name you give it to the message that gets logged, so these will actually show up like `(json_convert): JSON_Convert: [message]` in the log files